src/lib.rs

```diff
@@ -30,6 +25,7 @@ use serde::{Deserialize, Serialize};
 use serde_cbor::de::{Deserializer, IoRead};
 use serde_cbor::error::Error as CborError;
 use serde_cbor::ser::Serializer;
+use serde_cbor::ser::IoWrite;
```

Comment:
Nitpick: this could be folded into a single line, eg` use serde_cbor::ser::{IoWrite, Serializer}`.

Reply:
I always leave "Allow edits from maintainers" checked, so don't hesitate! But I just updated it for you.

Reply:
Thank you :-)

I prefer not to use that, it's better for author to acknowledge it and maybe learn something new, if it wasn't just oversight.